extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerBuildSupport.java

```diff
@@ -251,6 +266,49 @@ private boolean isInBuildDir(IPath sourceFullPath) {
         return Arrays.stream(sourceFullPath.segments()).anyMatch(segment -> segment.equals("build"));
     }
 
+    /**
+     * Get the JVM build target which has the highest version.
+     */
+    private JvmBuildTarget getJvmTargetWithHighestVersion(List<BuildTarget> buildTargets) throws CoreException {
+        List<JvmBuildTarget> jvmTargets = new LinkedList<>();
+        for (BuildTarget buildTarget : buildTargets) {
+            // https://build-server-protocol.github.io/docs/extensions/jvm#jvmbuildtarget
+            if ("jvm".equals(buildTarget.getDataKind())) {
+                String javaVersion = getString((Map) buildTarget.getData(), JAVA_VERSION);
+                if (StringUtils.isBlank(javaVersion)) {
+                    continue;
+                }
+
+                String javaHome = getString((Map) buildTarget.getData(), JAVA_HOME);
+                if (StringUtils.isBlank(javaHome)) {
+                    continue;
+                }
+                jvmTargets.add(new JvmBuildTarget(javaHome, javaVersion));
+            }
+        }
+
+        if (jvmTargets.isEmpty()) {
+            throw new CoreException(new Status(IStatus.ERROR, ImporterPlugin.PLUGIN_ID, "JVM target is unavailable."));
+        }
+
+        jvmTargets.sort((j1, j2) -> JavaCore.compareJavaVersions(j1.getJavaVersion(), j2.getJavaVersion()));
```

Comment:
It seems that you need to call `getEclipseCompatibleVersion(j1.getJavaVersion())` before comparing, because compareJavaVersions only recognizes 1.0 ~ 1.8 and 9.x.